app/src/main/java/org/simple/clinic/patient/LastBp.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.patient
+
+import org.threeten.bp.Instant
+import java.util.UUID
+
+data class LastBp(
```

Comment:
This data class can sit inside `PatientSearchResult` since its not being used anywhere else.